src/main/webapp/auth.html

```diff
@@ -0,0 +1,41 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Authentication Page</title> 
+    <link rel="stylesheet" href="style.css">
+  </head>
+ <body onload="googleSignIn()">
+    
+
+      <!-- Initialize firebase products -->
+  <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-app.js"></script>
+  <script defer src="https://www.gstatic.com/firebasejs/7.15.5/firebase-auth.js"></script>
+
+  <script>
+    // Your web app's Firebase configuration
+    var firebaseConfig = {
+      apiKey: "AIzaSyAGk_4TZiVf_TslWON5hNQDYVODuJifP7A",
+      authDomain: "com-sdk-dashboard-step-2020.firebaseapp.com",
+      databaseURL: "https://com-sdk-dashboard-step-2020.firebaseio.com",
+      projectId: "google.com:sdk-dashboard-step-2020",
+      messagingSenderId: "412575630040",
+      appId: "1:412575630040:web:824b2d31f4e1f06b100dad",   
+    };
+    // Initialize Firebase
+      firebase.initializeApp(firebaseConfig);
+
+    //Redirect to Google Sign In
+      function googleSignIn() {
+      var provider = new firebase.auth.GoogleAuthProvider();
+      firebase.auth().signInWithRedirect(provider).then(function(result) {
+          console.log(result);
```

Comment:
Reason for not using console.info & console.warn?

Reply:
I didn't know there was a difference between console.log and console.info. I just used what I was used to. Are those options better?